observability/cockpit/concepts.mdx

```diff
@@ -0,0 +1,174 @@
+
+</Concept>
+
+<Concept>
+
+## Loki Remote Write
+
+``Loki Remote Write`` is a feature that allows you to send metrics from almost any device to a Loki server.
+
+</Concept>
+
+<Concept>
+
+## LogQL
+
+LogQL is Grafana Loki’s language for querying logs. LogQL uses labels and operators for filtering.
+
+</Concept>
+
+<Concept>
+
+## Logs
+
+Logs provide a record of all events and errors that take place during the lifecycle of your resources. They represent an excellent source of visibility if you want to know when a problem occurred, or which events correlate with it.
+
+</Concept>
+
+<Concept>
+
+## Managed alerts
+
+Managed alerts are alert criteria that Scaleway defines for you. They allow you to receive alerts for behaviors that we deem abnormal on your products.
+
+</Concept>
+
+<Concept>
+
+## Managed dashboards
+
+A managed dashboard is a set of one or more panels organized into one or more rows where you can visualize your data.
+
+</Concept>
+
+<Concept>
+
+## Metric
+
+A metric is a lifecycle-related numerical representation of data (e.g. disk usage and CPU usage) measured over intervals of time. Metrics give you a bird's eye view of your infrastructure.
+
+</Concept>
+
+<Concept>
+
+## Monitoring
+
+Monitoring is a tool that allows you to watch and understand the state of your systems. Monitoring is based on gathering predefined sets of metrics or logs.
+
+</Concept>
+
+<Concept>
+
+## Prometheus Remote Write
+
+``Prometheus Remote Write`` is a feature that allows you to send your metrics from almost any device to a Prometheus server.
+
+</Concept>
+
+<Concept>
+
+## PromQL
+
+PromQL, short for Prometheus Querying Language, is the main way to query metrics within Prometheus. It is designed for building queries for graphs and alerts.
+
+</Concept>
+
+<Concept>
+
+## Receivers
+
+Receivers are hubs consisting of contacts points. You can associate one or several alerts with one or more receivers. This allows you to diversify your alerts.
+
+</Concept>
+
+<Concept>
+
+## Recording rules
+
+Recording rules allow you to precompute frequently needed or computationally expensive expressions and save their results as a new set of time series.
+
+</Concept>
+
+<Concept>
+
+## Samples
+
+A sample is a unique measuring point on a time series.
+
+</Concept>
+
+<Concept>
+
+## Time series
+
+A time series is a sequence of numerical data points in successive order, that tracks a sample over time.
+
+</Concept>
+
+<Concept>
+
+
+## Tokens
+
+Tokens are secret keys that allow you to grant other cloud providers’ services and platforms access to your Cockpit. You can generate new tokens and select token permissions such as push and query. Push and query allow you to send and fetch your data (metrics and logs) to and from your Cockpit. You can also choose to configure alerting and recording rules or set up the alert manager via your tokens.
```

Comment:
Tokens are secret keys that allow you to authenticate against your Cockpit endpoints (Metrics, Logs, Alerts).

You can generate new tokens and select token permissions such as push and query. Push and query allow you to send and fetch your data (metrics and logs) to and from your Cockpit.

You can also choose to configure alerting and recording rules or set up the alert manager using your tokens.